components/hal/esp32c6/include/hal/uart_ll.h

```diff
@@ -7,33 +7,27 @@
 // The LL layer for UART register operations.
 // Note that most of the register operations in this layer are non-atomic operations.
 
+
```

Comment:
looks like this file was rolled back to a previous version
can we keep latest one? function like `uart_ll_is_enabled` is necessary for PM